src/cplscheme/BaseCouplingScheme.hpp

```diff
@@ -27,7 +27,7 @@ class TXTTableWriter;
 } // namespace io
 
 namespace cplscheme {
-struct CouplingData;
+class CouplingData;
```

Comment:
@BenjaminRodenberg already did the same change in #1004 
Just fyi

Reply:
This shouldn't lead to conflicts as the change is identical.